01-main/packages/nuclear

```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+# they have no release, all are set as pre-release
+# using "latest" would return nothing
+get_github_releases "nukeop/nuclear"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f4)"
+    VERSION_PUBLISHED="$(echo "${URL}" | cut -d'/' -f8 | tr -d v)"
```

Comment:
this will prevent continuous update attempts until the next release - maybe a monthly chore :disappointed: 
`VERSION_PUBLISHED=${VERSION_PUBLISHED/fd06a3/0.6.17}`

Reply:
What about this? Changing PUBLISHED_VERSION
```
Reading package lists...
  [+] nuclear (0.6.17) has an update pending. 2023-02-19T162001Z-fd06a3 is available.
```
It will still offer an update but at least now user can see how old it is. Timestamp and also commit id.

```
$ ./deb-get show nuclear
  [+] Including local package nuclear
  [!] Please consider contributing back new entries, an issue (or raise a PR) directly at https://github.com/wimpysworld/deb-get/pulls
Nuclear
  Package:	nuclear
  Repository:	99-local
  Updater:	deb-get
  Installed1:	0.6.17
  Published:	2023-02-19T162001Z-fd06a3
  Architecture:	amd64
  Download:	https://github.com/nukeop/nuclear/releases/download/fd06a3/nuclear-fd06a3.deb
  Website:	https://nuclear.js.org/
  Summary:	Streaming music player that finds free music for you.
```